TrackletAlgorithm/AllStubInnerMemory.h

```diff
@@ -245,6 +245,19 @@ class AllStubInner : public AllStubInnerBase<ASType>
     data_.range(kASNegDiskMSB,kASNegDiskLSB) = negdisk;
   }
 
+#ifdef CMSSW_GIT_HASH
+  std::string getBitStr() const {
+    std::string str = decodeToBits(getR());
```

Comment:
@carriganm95 Do you know why you don't need to include globalFunctions.h in this file? I don't understand how the compiler knows what `decodeToBits` is without such an include.

Same question for TrackFitMemory.h and VMStubTEOuterMemoryCM.h.

Reply:
yeah I am not sure, TrackletParameter memory complained when I did not add it. My guess is that we have loaded other memory types in those Analyzers so that it already knows about globalFunctions? Either way I will add it to be safe.

Reply:
I have added the include statements

Reply:
> My guess is that we have loaded other memory types in those Analyzers so that it already knows about globalFunctions?

Yeah, that could be. In any case, I think it's good to include globalFunctions.h in each header file so you aren't dependent on how these are included in the analyzers.